src/docs/_sidebars/mainSidebar.md

```diff
@@ -18,8 +18,9 @@
 # Annotated Research Context:/docs/implementation/AnnotatedResearchContext.html
 ## User Journey:/docs/implementation/QuickStart_arc.html
 # ARC Commander:/docs/implementation/ArcCommander.html
-## QuickStart:/docs/implementation/QuickStart_arcCommander.html
 ## Manual: /docs/implementation/ArcCommanderManual/index.html
+## QuickStart:/docs/implementation/QuickStart_arcCommander.html
+## QuickStart (experts):/docs/implementation/QuickStart_arcCommander_expert.html
```

Comment:
Great, I love the QuickStart for experts as the other one can be quite overwhelming just at it is so long.

Reply:
That’s actually old and used to be at the bottom of the long quickstart. Point to discuss though: would we want such a no-explanation copy/paste type block?

Reply:
Good point but you anyway have to replace the place holders so you have to go through it line by line and most liekly this is not your first ARC... Fine for me as it reduces space and makes it very compact.

Reply:
I've hidden the expert version from the sidebar. It would still render (once merged) on the website.